pkg/mutation/system.go

```diff
@@ -48,10 +48,11 @@ func (s *System) Upsert(m types.Mutator) error {
 	toAdd := m.DeepCopy()
 
 	// Checking schema consistency only if the mutator has schema
+	var err error
 	if withSchema, ok := toAdd.(schema.MutatorWithSchema); ok {
-		err := s.schemaDB.Upsert(withSchema)
-		if err != nil {
-			return errors.Wrapf(err, "Schema upsert failed for %v", m.ID())
+		err2 := s.schemaDB.Upsert(withSchema)
```

Comment:
Looking at this function call, I'm not sure we want to report schema conflicts as an error, given the upsert now actually occurs and the system is working correctly.

Errors should be returned when something abnormal happened such that a re-reconcile may be in order.

Also, there are other possible errors that a schema Upsert can incur.

Reply:
We should not submit code with broken behavior either. If we kept unwind, this would be fine. As it is, this doesn't work.

We'll need to either re-add unwind, add the missing functionality, or figure out some third way forward that keeps HEAD functional.

Reply:
The desired behavior is unclear, so I'm setting up a meeting to figure out.

Reply:
Per meeting, I've changed System to manually remove the offending mutator on error. This preserves the existing behavior and is now validated via unit test.